backstory/src/main/java/com/google/sps/data/perspective/PerspectiveServiceClient.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.perspective;
+
+import au.com.origma.perspectiveapi.v1alpha1.PerspectiveAPI;
+import au.com.origma.perspectiveapi.v1alpha1.models.AnalyzeCommentRequest;
+import au.com.origma.perspectiveapi.v1alpha1.models.AnalyzeCommentRequest.Builder;
+import au.com.origma.perspectiveapi.v1alpha1.models.AnalyzeCommentResponse;
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeScore;
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+import au.com.origma.perspectiveapi.v1alpha1.models.ContentType;
+import au.com.origma.perspectiveapi.v1alpha1.models.Entry;
+import au.com.origma.perspectiveapi.v1alpha1.models.RequestedAttribute;
+import au.com.origma.perspectiveapi.v1alpha1.models.Score;
+
+/**
+ * A service client to access the Perspective API 
+ */
+public class PerspectiveServiceClient {
+  public static PerspectiveAnalysis analyze(PerspectiveAPI perspective, AttributeType[] types, String text) {
```

Comment:
Nit: renaming `types` -> `attributeTypes` throughout this method would improve clarity (type is fairly generic so when I was reading the method I kept wondering, what is this a type of?)

Reply:
That makes sense. I will change it to clarify it. Thank you.

Reply:
Done